package.json

```
@@ -28,7 +28,7 @@
   },
   "keywords": [],
   "author": "@djh",
-  "license": "MIT",
+  "license": "Apache 2.0",
```

```suggestion
  "license": "Apache-2.0",
```

This must be a valid value from the [SPDX License List](https://spdx.org/licenses/)

---

🤦‍♀️